bootstrapper/misc/miscNg2Context.ts

```diff
@@ -2,14 +2,18 @@ import { Component, ViewChild } from '@angular/core';
 import { Observable, Subject } from 'rxjs';
 
 import { BusyComponent } from '../../source/components/busy/busy';
+import { RatingBarComponent } from '../../source/components/ratingBar/ratingBar';
+import { SpinnerComponent } from '../../source/components/inputs/index';
 
 @Component({
 	selector: 'tsMiscNgContext',
 	template: require('./miscNg2Context.html'),
-	directives: [BusyComponent],
+	directives: [BusyComponent, RatingBarComponent, SpinnerComponent],
 })
 export class MiscNgContextBootstrapper {
 	@ViewChild('busy2') busy2: BusyComponent;
+	width: number = 100;
+	value: number = 0;
 
 	wait(): Observable<void> {
 		const subject: Subject<void> = new Subject<void>();
```

Comment:
Yeah, it doesn't seem as egregious with numbers, strings and booleans.  I still prefer to see those types explicitly declared.  I'm mainly in favor of dropping the explicit typing on complex objects where the type would appear on both sides of the equal sign.